tests/Model/EditableCustomRuleTest.php

```diff
@@ -47,4 +47,24 @@ public function testToggleDisplayText()
         $this->assertSame('addClass("hide")', $rule1->toggleDisplayText('show'));
         $this->assertSame('removeClass("hide")', $rule1->toggleDisplayText('hide'));
     }
+
+    /**
+     * Test that methods are returned for manipulating the presence of the "hide" CSS class depending
+     * on whether the field should be hidden or shown
+     */
+    public function testValidateAgainstFormData()
```

Comment:
At first glance, this doesn't appear to be covering all the cases in the switch statement.

Reply:
I've added extra test coverage.